tests/console.py

```diff
@@ -6,6 +6,7 @@
 from tests.support import Home, Pet, User, database
 
 logging.basicConfig(level=logging.DEBUG)
+logging.getLogger("parso").setLevel(logging.ERROR)
```

Comment:
What does this do?

Reply:
I was getting a lot of errors from parso cluttering my `bin/console`. Were you not seeing them?

Reply:
Okay, no I didn't notice them, but this is fine then.